### PR TITLE
Modify Readme to be more clear about Gitter Channels

### PR DESCRIPTION
**Pull request by PFCKrutonium**
Makes it clear which gitter channel to go to for help. I've noticed a lot of people looking for help end up in the wrong channel, so this should help prevent that.

### CODE REVIEW

**Comment by IntelOrca** [comment]
You appear to have removed the last change that got added to the readme.

Can you also make it look nicer? Maybe put the badges in a table?

**Comment by PFCKrutonium** [comment]
O.o I don't see how I managed to do that. Let me just fix that.

**Comment by Gymnasiast** [comment]
I'd also want to tweak the descriptions a bit. (But that won't be today, too tired right now.)

**Comment by PFCKrutonium** [comment]
@Gymnasiast If you mention how you want them tweaked, I can try to work that in.

File: readme.md [hunk 1]
```diff
@@ -8,11 +8,15 @@ An open-source re-implementation of RollerCoaster Tycoon 2. A construction and m
...
 
 ### Chat
-English:<br />
-[![Gitter](https://img.shields.io/badge/gitter-general-blue.svg)](https://gitter.im/OpenRCT2/OpenRCT2/non-dev)<br />
-[![Gitter](https://img.shields.io/badge/gitter-development-yellowgreen.svg)](https://gitter.im/OpenRCT2/OpenRCT2)<br />
-Nederlands:<br />
-[![Gitter](https://img.shields.io/badge/gitter-general-blue.svg)](https://gitter.im/OpenRCT2/OpenRCT2/Nederlands)
+If you want to help *make* the game, join the Developer Channel for your Language.
+
+If you need help, or want to talk to the developers, or just want to stay up to date, join the Non Developer channels.
```
Comment (Gymnasiast) [inline readme.md]: If you need help, or want to talk to the developers, or just want to stay up to date, join the non-developer channels.

File: readme.md [hunk 2]
```diff
@@ -8,11 +8,15 @@ An open-source re-implementation of RollerCoaster Tycoon 2. A construction and m
 | **develop** | [![AppVeyor](https://ci.appveyor.com/api/projects/status/7efnemxhon6i5n34/branch/develop?svg=true)](https://ci.appveyor.com/project/IntelOrca/openrct2-ject9) | [![Travis CI](https://travis-ci.org/OpenRCT2/OpenRCT2.svg?branch=develop)](https://travis-ci.org/OpenRCT2/OpenRCT2) | [![OpenRCT2.org](https://img.shields.io/badge/develop-v0.0.8-blue.svg)](https://openrct2.org/downloads/develop/latest) |
 
 ### Chat
-English:<br />
-[![Gitter](https://img.shields.io/badge/gitter-general-blue.svg)](https://gitter.im/OpenRCT2/OpenRCT2/non-dev)<br />
-[![Gitter](https://img.shields.io/badge/gitter-development-yellowgreen.svg)](https://gitter.im/OpenRCT2/OpenRCT2)<br />
-Nederlands:<br />
-[![Gitter](https://img.shields.io/badge/gitter-general-blue.svg)](https://gitter.im/OpenRCT2/OpenRCT2/Nederlands)
+If you want to help *make* the game, join the Developer Channel for your Language.
```
Comment (Gymnasiast) [inline readme.md]: If you want to help *make* the game, join the developer channel.

**Comment by Gymnasiast** [comment]
You already tweaked them a great deal in the direction I wanted. I left line note to address to use of capitals and to reflect that Nederlands is a non-dev channel.

File: readme.md (hunk 2 above)
Comment (IntelOrca) [inline readme.md]: develop?

**Comment by IntelOrca** [comment]
Add localisation too

**Comment by PFCKrutonium** [comment]
Hm, any issues with me imbedded the project logo at the top of the readme? Also, does anyone know how to center an image in  markdown?

**Comment by IntelOrca** [comment]
No issues, but make sure it isn't obtrusive. A screenshot would also be nice. Right aligned to the page if possible.

Maybe see how https://github.com/CorsixTH/CorsixTH do it?

**Comment by PFCKrutonium** [comment]
Alright, I'll see what I can do.

_Sent from my LGE Nexus 5 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by PFCKrutonium** [comment]
Can be merged if agreeable.

**Comment by IntelOrca** [comment]
I think the screenshot should be smaller and be of an OpenRCT2.org group park that shows more custom content and nice architecture.

**Comment by PFCKrutonium** [comment]
Any Suggestions for which park in particular?

**Comment by IntelOrca** [comment]
Nope

**Comment by rwjuk** [comment]
Nice screenshot choice IMO!

**Comment by Gymnasiast** [comment]
There seems to be an error in the placement of one of the Asian walls at the top of the lift hill.

**Comment by PFCKrutonium** [comment]
So choose another screenie?

**Comment by Gymnasiast** [comment]
Nah, just fix that one rogue wall.

**Comment by rwjuk** [comment]
Just for future reference, if you're *only* changing text files (e.g. the readme, changelog, etc.), can you put `[ci skip]` in the commit message body so that the continuous integration systems don't run unnecessarily?

**Comment by PFCKrutonium** [comment]
I forgot about that.

**Pull request merged**